src/metabase/lib/util.cljc

```diff
@@ -560,3 +561,8 @@
   (when (map? query)
     (keyword (some #(get query %)
                    [:lib/type :type "lib/type" "type"]))))
+
+(mu/defn referenced-field-ids :- [:maybe [:sequential ::lib.schema.id/field]]
+  "Find all the integer field IDs in `coll`, Which can arbitrarily be anything that is part of MLv2 query schema."
```

Comment:
typo
```suggestion
  "Find all the integer field IDs in `clause`, Which can arbitrarily be anything that is part of MLv2 query schema."
```

I'm not sure this is the best place to put this function. It's coupled to the internal structure of a `:mbql.clause/field` as defined here:
https://github.com/metabase/metabase/blob/cf29360f37a4f2cb76f1d61fc8f1642f2e1a8e61/src/metabase/lib/schema/ref.cljc#L51

So maybe it makes sense to put it there

Reply:
those lib.schema.* ns are all about schema. I don't think anyone will expect to find a helper in there 🤔

Reply:
true, that's unfortunate because that's where the code is coupled to. maybe if it were in `metabase.lib.field` it would be slightly more discoverable? maybe a good q for qpd

Reply:
I agree that `metabase.lib.schema.*` is the wrong place for helper functions.

This is about field IDs but that doesn't mean it belongs in `lib.field`... I'm not sure there's really a better place than `util`.